### PR TITLE
Feature: 166 Refactor SVGs from Components to Files

### LINKED ISSUE #166: 📝 Refactor SVGs from Components to Files
## 🎯 Describe the task:
This task involves locating all SVGs currently embedded within our components and refactoring them into individual SVG files. This can enhance the organization and readability of our codebase, leading to potentially improved rendering performance.

## ℹ️ Additional Context:
Storing SVGs as separate files can improve the maintainability of our project. It'll be easier to manage, update, and reuse SVGs. Additionally, it may improve the rendering performance of our website as the browser can cache the SVG file.

## ✅ Definition of Done (DoD):
- All SVGs in components are identified and successfully refactored into individual SVG files.
- Refactored SVGs are correctly displayed in the application.
- SVG file storage is organized and files are easily accessible.
- There is no negative impact on the existing functionality of the web application.
- The changes pass all existing tests.
- The implementation has been reviewed and approved by the team.

### PR DESCRIPTION
**Pull request by stroich**
## What type of PR is this? (select all that apply)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 🚧 Breaking Change
- [ ] 🧑‍💻 Code Refactor
- [ ] 📝 Documentation Update

## Related Tickets & Documents

- For each identified SVG image, created a separate file with a corresponding name.
- Saved SVG images in SVG  format to ensure compatibility across a wide range of browsers.
- Updated component code to utilize references to the created files instead of embedded SVG images.

- Related Issue #166 
- Closes #166

## Added/updated tests?

- [x] 👌 Yes
- [ ] 🙅‍♂️ No, because they aren't needed
- [ ] 🙋‍♂️ No, because I need help

### CODE REVIEW

File: src/icons/js-frontend-icon.tsx [hunk 1]
```diff
@@ -1,32 +1,6 @@
...
-        y2="0.971646"
-        gradientUnits="userSpaceOnUse">
-        <stop stopColor="white" stopOpacity="0.85"></stop>
-        <stop offset="1" stopColor="white" stopOpacity="0.35"></stop>
-      </linearGradient>
-    </defs>
-  </svg>
-);
+export const JsFrontendIcon = () => {
+  return <Image src={jsFrontend} alt="JsFrontend icon" />;
```
Comment (andron13) [inline src/icons/js-frontend-icon.tsx:5]: Could you please clarify where the { width = '48', height = '36' } code snippet has been relocated? The dimensions of the image seem to be missing.

**Review by andron13 — APPROVED** [review]
With logos without dimensions, it seems like a risky practice to me.

File: src/icons/instagram.tsx [hunk 2]
```diff
@@ -1,31 +1,10 @@
...
-      strokeWidth="2"
-      stroke="rgb(0,0,0)"
-      strokeLinecap="round"
-      strokeLinejoin="round"
-      strokeDasharray=""></path>
-  </svg>
-);
+export const InstagramIcon = () => {
+  return (
+    <div className="icon-surface">
```
Comment (ramzes-84) [inline src/icons/instagram.tsx:6]: [We decided](https://github.com/rolling-scopes/site/blob/main/readme/semantic-webaccessibility.md) not to use `div`s, so can we use `figure` here?

File: src/icons/linkedIn.tsx [hunk 3]
```diff
@@ -1,28 +1,10 @@
...
-      stroke="black"
-      strokeWidth="2"
-      strokeLinecap="round"
-      strokeLinejoin="round"
-    />
-  </svg>
-);
+export const LinkedInIcon = () => {
+  return (
+    <div className="icon-surface">
```
Comment (ramzes-84) [inline src/icons/linkedIn.tsx:6]: Can we use `figure` here?

File: src/icons/telegram.tsx [hunk 4]
```diff
@@ -1,17 +1,10 @@
...
-      fill="black"
-    />
-  </svg>
-);
+export const TelegramIcon = () => {
+  return (
+    <div className="icon-surface">
```
Comment (ramzes-84) [inline src/icons/telegram.tsx:6]: Can we use `figure` here?

File: src/icons/youtube.tsx [hunk 5]
```diff
@@ -1,28 +1,10 @@
...
-    <defs>
-      <clipPath id="clip0_20960_32044">
-        <rect width="24" height="24" fill="white" transform="translate(16.6665 16.0684)" />
-      </clipPath>
-    </defs>
-  </svg>
-);
+export const YouTubeIcon = () => {
+  return (
+    <div className="icon-surface">
```
Comment (ramzes-84) [inline src/icons/youtube.tsx:6]: Can we use `figure` here?

**Review by ramzes-84 — APPROVED** [review]
Great job!

File: src/icons/instagram.tsx (hunk 2 above)
Comment (stroich) [reply src/icons/instagram.tsx]: Done

**Review by ramzes-84 — APPROVED** [review]

File: src/icons/js-frontend-icon.tsx (hunk 1 above)
Comment (stroich) [reply src/icons/js-frontend-icon.tsx]: This component has been removed because it is not used on the project.

File: src/icons/logo.tsx [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import logo from '@/assets/svg/rss-logo.svg';
+import Image from '@/features/image';
+
+interface LogoProps {
+  type: 'navbar' | 'footer';
+}
+
+export const Logo = ({ type }: LogoProps) => {
+  return <Image className={type !== 'navbar' ? 'white-logo' : ''} src={logo} alt="logo" />;
```
Comment (Quiddlee) [inline src/icons/logo.tsx:9]: Please add `classnames` package, and change to this
https://www.npmjs.com/package/classnames

```tsx
className={classNames({
  'white-logo': type === 'footer',
)}
```

Comment (stroich) [reply src/icons/logo.tsx]: Done

**Review by Quiddlee — APPROVED** [review]

**Pull request merged**